kube/ingress.yml

```diff
@@ -2,20 +2,24 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
+  labels:
+    stable.k8s.psg.io/kcm.class: default
   annotations:
-    ingress.kubernetes.io/secure-backends: "true"
+    ingress.kubernetes.io/force-ssl-redirect: "true"
+    kubernetes.io/ingress.class: "nginx-external"
+    stable.k8s.psg.io/kcm.provider: http
```

Comment:
Why http validation? dns validation would work fine if you change the domain you're deploying to.

Reply:
If we keep developer-induction.notprod.homeoffice.gov.uk domain we will need to have kcm provider as http since this is will not be in our route53 table, but yeah could also have secure-backend.